RecoVertex/BeamSpotProducer/src/PVFitter.cc

```diff
@@ -350,9 +350,10 @@ bool PVFitter::runFitter() {
     TH1F *h1PVz = (TH1F*) hPVx->ProjectionY("h1PVz", 0, -1, "e");
 
     //Use our own copy for thread safety
-    TF1 gausx("localGausX","gaus");
-    TF1 gausy("localGausY","gaus");
-    TF1 gausz("localGausZ","gaus");
+    // also do not add to global list of functions
+    TF1 gausx("localGausX","gaus",false);
```

Comment:
@Dr15Jones The 3rd argument in the case of TF1 is a different type than in the case of a TFormula.  For TF1, it is a TF1::EAddToList
```
   // Add to list behavior
   enum class EAddToList {
      kDefault,
      kAdd,
      kNo
   };
```
so this code actually 'still' request the addition to the list (i.e. kDefault)